### PR TITLE
Run kubeconform on manifests.

### PR DESCRIPTION
**Pull request by porridge**

### CODE REVIEW

File: deploy/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/.gotools/
```
Comment (janisz) [inline deploy/.gitignore:1]: do we need first `/`?
```suggestion
.gotools/
```

File: make/gotools.mk [hunk 2]
```diff
@@ -0,0 +1,130 @@
+# gotools.mk
+# Simplified installation & usage of Go-based tools
+# Copied from https://github.com/stackrox/stackrox/blob/master/make/gotools.mk
```
Comment (janisz) [inline make/gotools.mk:3]: :heart:

File: deploy/.gitignore (hunk 1 above)
Comment (porridge) [reply deploy/.gitignore:1]: Leading `/` was intentional, means "just in this dir". But thinking about it, it makes sense to remove ignore it everywhere...

Comment (porridge) [reply deploy/.gitignore]: Moved to root.

**Pull request merged**